odl/operator/operator.py

```diff
@@ -237,6 +461,10 @@ def __init__(self, domain, range, linear=False):
                 raise TypeError('range {!r} not a `LinearSpace` or `Field` '
                                 'instance.'.format(range))
 
+    def _call(self, *args, **kwargs):
```

Comment:
Perhaps

`def _call(self, x, out=None, **kwargs)` or similar? Maybe a pointer (See also) to where more information on this can be found.